src/graphql/Admin/Queries/getStatistics.ts

```diff
@@ -0,0 +1,16 @@
+import { ApplicantRepository } from "$models/Applicant";
+import { CompanyRepository } from "$models/Company";
+import { JobApplicationRepository } from "$models/JobApplication";
+import { OfferRepository } from "$models/Offer";
+import { GraphQLStatistics } from "../Types/GraphQLStatistics";
+
+export const getStatistics = {
+  type: GraphQLStatistics,
+  resolve: async () => ({
+    amountOfStudents: await ApplicantRepository.countStudents(),
+    amountOfGraduates: await ApplicantRepository.countGraduates(),
+    amountOfCompanies: await CompanyRepository.countCompanies(),
+    amountOfJobApplications: await JobApplicationRepository.countJobApplications(),
+    amountOfOffers: await OfferRepository.countCurrentOffers()
```

Comment:
creo que deberían tener "Approved" en el nombre, también (esto mas subjetivo) yo haria algo como "get approved student count" porque suena mas a query a la bd

Reply:
donde dice "amount of X" lo cambiaria por "X count"